content/business-models/source-available.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Source Available"
+date: 2019-06-04
+weight: 1
+---
+
+While copyleft licenses (e.g., [Parity](https://paritylicense.com/), 
+[AGPL](https://en.wikipedia.org/wiki/Affero_General_Public_License)) were 
+originally created to protect intellectual property of a community, they have 
+not served well in all cases to protect communities from possibly being taken 
+advantage of by a larger corporate entity. For example, a copyleft license may 
+allow for use of open source software by a company, and charging for services 
+afforded by that software. While industry might give back to the community by 
+way of contributions, this is not always the case. In these unfortunate cases, 
+the communities get defensive, and the overall dynamic encourages less openness
+and sharing because of that. Specifically, we see from this practice:
+
+ 1. As a defensive mechanism, software becomes proprietary to protect itself.
+ 2. New open source licenses are developed as another means of protection.
+
+The second defense mechanism has resulted in discussion about development
+of "source available" licenses that dictate terms under which an industry
+entity can deploy infrastructure that profits from a piece of software. 
+For more background on the development, see 
+[this article](https://techcrunch.com/2019/05/30/lack-of-leadership-in-open-source-results-in-source-available-licenses/).
+
+Interestingly, this initiative has resulted directly from a lack of 
+initiative on the part of some industry entities to contribute back. It serves
+as a powerful example for how soft skills, culture, and supporting a 
+software community are essential factors that when neglected, can lead 
```

Comment:
Bump neglect to the beginning to enhance readability by obviously negating them preemptively instead of forcing cognitive load via revisionary sentence structure.
```
It serves as a powerful example for how neglecting soft skills, culture, and supporting a software community are essential factors can lead to a less open landscape.
```

Reply:
Aye.